Formula/fluent-bit.rb

```diff
@@ -1,8 +1,8 @@
 class FluentBit < Formula
   desc "Data Collector for IoT"
   homepage "https://github.com/fluent/fluent-bit"
-  url "https://github.com/fluent/fluent-bit/archive/v1.8.1.tar.gz"
-  sha256 "a284d803ec9203be69b7c4ff9febb33b8c938b640f0cc7cbcf6f9223e5de4fb5"
+  url "https://github.com/fluent/fluent-bit/archive/v1.8.3.tar.gz"
```

Comment:
Not the latest release on GitHub (1.8.2 is).

Reply:
✅